### PR TITLE
cleanup: remove unneeded replaces

### PR DESCRIPTION
**Pull request by aalu1418**
cleaning up replaces to match: https://github.com/smartcontractkit/chainlink/pull/13980

### CODE REVIEW

File: integration-tests/go.mod [hunk 1]
```diff
@@ -492,35 +492,4 @@ replace (
 
 	// type func(a Label, b Label) bool of func(a, b Label) bool {…} does not match inferred type func(a Label, b Label) int for func(a E, b E) int
 	github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.47.2-0.20231010075449-4b9c19fe5510
```
Comment (jmank88) [inline integration-tests/go.mod:494]: This is avoidable now too, but can be separate PR.

**Review by jmank88 — DISMISSED** [review]

**Review by jadepark-dev — APPROVED** [review]
